src/keymon/keymon.c

```diff
@@ -174,13 +174,6 @@ void shutdown(void)
     system_clock_save();
     sync();
 
-    if (DEVICE_ID == MIYOO283) {
-        reboot(RB_AUTOBOOT);
-    }
-    else if (DEVICE_ID == MIYOO354) {
-        system("poweroff");
-    }
-
```

Comment:
We need to put `system("shutdown")` here as well.

Reply:
The flag is not enough ? 
https://github.com/OnionUI/Onion/blob/6d1cb0d7f111ca360aaf3e91611a9237adb42bd2/src/common/system/system.h#L29
It already shutdown like that thanks to runtime.sh 🤔

Reply:
No, this function is used to force shutdown (when runtime.sh is locked by a process not terminating)